src/ribbon/art_aui.cpp

```diff
@@ -233,34 +247,38 @@ void wxRibbonAUIArtProvider::SetColourScheme(
     // TODO: Remove next line once this provider stops piggybacking MSW
     wxRibbonMSWArtProvider::SetColourScheme(primary, secondary, tertiary);
 
-#define LikePrimary(luminance) \
-    wxRibbonShiftLuminance(primary_hsl, luminance ## f).ToRGB()
-#define LikeSecondary(luminance) \
-    wxRibbonShiftLuminance(secondary_hsl, luminance ## f).ToRGB()
+    const auto LikePrimary = [&primary_hsl](const float luminance)
```

Comment:
We don't use toplevel const in functions declarations, i.e. this should be
```suggestion
    const auto LikePrimary = [&primary_hsl](float luminance)
```

I also wonder if we really need to pass the HSL colour by reference, I'd just copy it for simplicity. But this is even more minor, of course.

Reply:
Got it, will make those changes.